pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -28,7 +29,7 @@ let
 
   common = { version
     , sha256
-    , extraPatches ? []
+    , extraPatches ? zfsPatches
```

Comment:
How is this easier than `linuxPackages.zfs.overrideAttrs (old: { patches = old.patches ++ [ ./foo.patch ]; })`?

Reply:
Thanks for the suggestion. This change is created in 2019, and I overlooked `linuxPackages.extend` exists at that time. Now I could apply my patch with your suggestion:

```
boot.kernelPackages = pkgs.linuxPackages.extend (self: super: {
  zfs = super.zfs.overrideAttrs (old: { patches = old.patches ++ [ ./foo.patch ]; })
});
```